isotovideo

```diff
@@ -72,11 +115,12 @@ Getopt::Long::Configure("no_ignore_case");
 use OpenQA::Isotovideo::Interface;
 use OpenQA::Isotovideo::Runner;
 use OpenQA::Isotovideo::Utils qw(git_rev_parse spawn_debuggers handle_generated_assets);
+use Mojo::File qw(path);
 
 my %options;
 
 # global exit status
-my $return_code = 1;
+my $return_code = EXIT_STATUS_ERR;
```

Comment:
Standard Perl convention is for global variables to be all uppercase, like `$RETURN_CODE`. It's not a rule we've been very good about following in openQA though.

Reply:
Renamed them to `$RETURN_CODE`